src/page_body/structure/blocks/page_block_frames.pr

```diff
@@ -2,11 +2,11 @@
 
 {[ let variant = block.variantKey /]}
 
+{[ log block.properties /]}
 {* Configure tile to show natural height if needed *}
-{[ let forceNaturalHeight = (block.properties.alignment === "FrameHeight" || block.properties.layout == "C1" || block.properties.layout == "C1_75") /]}
-
```

Comment:
I'm not sure as to why we were defaulting to `forceNaturalHeight` with 1 column layouts, maybe you can help me out with this? @honzatmn